### PR TITLE
Add Interaction submission animation (DEV-177)

### PR DESCRIPTION
**Pull request by Davit-BetterAngels**
DEV-177

submit screen if user submitted after creating a note for the first time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
 <br>
 <br>
-
```
Comment (vecchp) [inline README.md:23]: This breaks markdown formatting.

File: apps/betterangels/eas.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
       "ios": {
         "ascAppId": "6468135540"
       },
-     "android": {
+      "android": {
```
Comment (vecchp) [inline apps/betterangels/eas.json:46]: why is this file updated?

File: apps/betterangels-backend/common/static/assets/images/icon.png
Comment (vecchp) [inline apps/betterangels-backend/common/static/assets/images/icon.png:1]: Is there a way we can consolidate all these assets? I think this is the 4th version of this icon in the repo.

**Review by vecchp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Davit-BetterAngels) [reply README.md:23]: haven't touched. everything was merged from main branch. and accepted all incoming.

File: apps/betterangels/eas.json (hunk 2 above)
Comment (Davit-BetterAngels) [reply apps/betterangels/eas.json:46]: haven't touched. everything was merged from main branch. and accepted all incoming.

File: apps/betterangels-backend/common/static/assets/images/icon.png
Comment (Davit-BetterAngels) [reply apps/betterangels-backend/common/static/assets/images/icon.png:1]: haven't touched backend app. maybe @mikefeldberg can answer why icon is here

**Review by mikefeldberg — CHANGES_REQUESTED** [review]
It looks like this branch was checked out from `main` before we reset githistory, and then the "new" `main` was merged into this branch. This PR should be closed and reopened from a new branch with just the affected files. 

image

**Comment by mikefeldberg** [comment]
replaced by #455

**Pull request closed without merging**